node/src/types/block.rs

```diff
@@ -1265,13 +1266,38 @@ impl Display for BlockSignatures {
 
 /// A proto-block after execution, with the resulting post-state-hash.  This is the core component
 /// of the Casper linear blockchain.
-#[derive(DataSize, Clone, Debug, PartialOrd, Ord, PartialEq, Eq, Hash, Serialize, Deserialize)]
+#[derive(DataSize, Clone, Debug, PartialOrd, Ord, PartialEq, Eq, Hash, Serialize)]
 pub struct Block {
     hash: BlockHash,
     header: BlockHeader,
     body: BlockBody,
 }
 
+/// A temporary copy of a block that has not been validated yet.
+#[derive(Debug, Deserialize)]
+pub(crate) struct UnverifiedBlock {
```

Comment:
Does it have to be `pub(crate)`? Looks like just an implementation detail of `Block` deserialization, so it should be fine for it to be private to this module, shouldn't it?

Reply:
Fixed in https://github.com/casper-network/casper-node/pull/1928/commits/ec67884eb3bf804f86c663ab5f0982a81b8b7cbf.